src/Symfony/Component/Mime/Resources/bin/update_mime_types.php

```diff
@@ -66,6 +64,27 @@
 $map = array_replace_recursive($current, $new);
 ksort($map);
 
+// remove some extensions from the map
+$removeExtensionByMimeType = [
+    'application/octet-stream' => 'msi',
```

Comment:
sorry I missed telling that my note about this should apply to all extensions, not only msi